getting_started/workflow/project_setup/version_control_system.rst

```diff
@@ -0,0 +1,31 @@
+.. _doc_version_contro_system:
+
+Version Control System
+======================
+
+Introduction
+------------
+
+Godot supports the use of version control systems, they can be
+setup or shut down in the editor under Project, Version Control.
```

Comment:
Documentation usually uses **Project > Version Control** format, wrapped in double asterisks.

Also, there is no need to write this as a continuous sentence. It's better to expand it a bit, and write as complete sentences. I also think it would be better to combine this part with the later part. So you end up with three sentences before the screenshot:
* "Godot has support".
* "You need a plugin".
* "You can configure it in settings".
* Screenshot
* Next section